eidpdata/src/main/java/uk/ac/ucl/eidp/data/jaxb/StatementGenerator.java

```diff
@@ -185,19 +171,19 @@ public void setSize(Integer size) {
       this.size = size;
     }
 
-//        public String getFormat() {
-//            return format;
-//        }
-//
-//        public void setFormat(String format) {
-//            this.format = format;
-//        }
+    //public String getFormat() {
+    //  return format;
+    //}
+    //
+    //public void setFormat(String format) {
+    //  this.format = format;
+    //}
 
     public Integer getSqlType() {
       TableFieldTypeType fieldType = TableFieldTypeType.fromValue(type);
       switch (fieldType) {
       case STRING :
-        if (getSize() > 254) {
+        if (getSize() > VARCHAR_LIMIT) {
```

Comment:
\[Checkstyle\] WARNING: 'if' have incorrect indentation level 8, expected level should be 10\.